src/components/Content/SideBar.jsx

```diff
@@ -0,0 +1,46 @@
+/* eslint react/no-unused-prop-types: 0 */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Links from '../Links/Links';
+
+const linksData = {
+  title: 'Program Documents',
+  links: [
+    {
+      title: 'Degree Requirements',
+      href: 'https://edx.org',
+    },
+    {
+      title: 'Student Record',
+      href: 'https://edx.org',
+    },
+  ],
+};
+
+class SideBar extends Component {
+  state = {};
```

Comment:
State is unused in this component, so no need to include it here.

Reply:
Removed